packages/plugins/plugin-navtree/src/experimental/Tree.tsx

```diff
@@ -29,7 +29,7 @@ export const IconButton = ({
   // TODO(burdon): Density aware.
   return (
     <div className={mx('flex w-6 h-6 items-center justify-center select-none', classNames)} onClick={onClick}>
-      <Icon size={size} className='cursor-pointer' {...props} />
```

Comment:
this was correct?

Reply:
i decided to leave this alone since it's experimental and has the duotone issue. so i'll fix in another pass when we figure out if/how we want to handle that.